src/uri.rs

```diff
@@ -34,6 +34,12 @@ impl Uri {
     pub fn amount_sats(&self) -> Option<u64> {
         self.0.amount.map(|x| x.to_sat())
     }
+    pub fn label(&self) -> Option<String> {
+        self.0.label.clone().map(|x| String::try_from(x).unwrap())
+    }
+    pub fn message(&self) -> Option<String> {
+        self.0.message.clone().map(|x| String::try_from(x).unwrap())
+    }
```

Comment:
If we unwrap the program will panic, so we want to do a graceful conversion to `Option` instead.
```suggestion
    pub fn label(&self) -> Option<String> {
        self.0.label.clone().and_then(|x| String::try_from(x).ok())
    }
    pub fn message(&self) -> Option<String> {
        self.0.message.clone().and_then(|x| String::try_from(x).ok())
    }
```